### ISSUE TITLE
Whitenoise collectstatic fails due to nonexistent file

### ISSUE THREAD

**Issue by michaelhays** [marcgibbons/django-rest-swagger#489]
When pushing to a Heroku app using `django-rest-swagger==2.0.1`, I encountered the following error:

```
$ python manage.py collectstatic --noinput
  Post-processing 'rest_framework_swagger/css/style.css' failed!
  Traceback (most recent call last):
    File "manage.py", line 10, in <module>
      execute_from_command_line(sys.argv)
    File "/app/.heroku/python/lib/python3.5/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
      utility.execute()
    File "/app/.heroku/python/lib/python3.5/site-packages/django/core/management/__init__.py", line 345, in execute
      self.fetch_command(subcommand).run_from_argv(self.argv)
    File "/app/.heroku/python/lib/python3.5/site-packages/django/core/management/base.py", line 348, in run_from_argv
      self.execute(*args, **cmd_options)
    File "/app/.heroku/python/lib/python3.5/site-packages/django/core/management/base.py", line 399, in execute
      output = self.handle(*args, **options)
    File "/app/.heroku/python/lib/python3.5/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 176, in handle
      collected = self.collect()
    File "/app/.heroku/python/lib/python3.5/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 128, in collect
      raise processed
  whitenoise.storage.MissingFileError: The file 'rest_framework_swagger/images/logo.png' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x7faa1694f3c8>.
  The CSS file 'rest_framework_swagger/css/style.css' references a file which could not be found:
    rest_framework_swagger/images/logo.png
  Please check the URL references in this CSS file, particularly any
  relative paths which might be pointing to the wrong location.

  Error while running '$ python manage.py collectstatic --noinput'.
  See traceback above for details.
```

Similarly, running `python manage.py collectstatic` on my local system:

```
...
Post-processed 'rest_framework_swagger/css/screen.css' as 'rest_framework_swagger/css/screen.991d73c77203.css'
Post-processed 'rest_framework_swagger/css/reset.css' as 'rest_framework_swagger/css/reset.721a281d7112.css'
Post-processing 'rest_framework_swagger/css/style.css' failed!

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 176, in handle
    collected = self.collect()
  File "~/.virtualenvs/drfw/lib/python3.4/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 128, in collect
    raise processed
whitenoise.storage.MissingFileError: The file 'rest_framework_swagger/images/logo.png' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x7fc007f40390>.

The CSS file 'rest_framework_swagger/css/style.css' references a file which could not be found:
  rest_framework_swagger/images/logo.png

Please check the URL references in this CSS file, particularly any
relative paths which might be pointing to the wrong location.
```

Looking in the [images directory](https://github.com/marcgibbons/django-rest-swagger/tree/master/rest_framework_swagger/static/rest_framework_swagger/images), it doesn't appear that `logo.png` (referenced on [line 5 of style.css](https://github.com/marcgibbons/django-rest-swagger/blob/master/rest_framework_swagger/static/rest_framework_swagger/css/style.css#L5)) or `shield.png` (referenced on [line 67 of style.css](https://github.com/marcgibbons/django-rest-swagger/blob/master/rest_framework_swagger/static/rest_framework_swagger/css/style.css#L67)) exist. I removed these two lines and `python manage.py collectstatic` finished without an error.

I replicated the issue on another project that uses version 2.0.1 of django-rest-swagger and had the same error.

**Comment by rloomans** [comment]
+1

#487 fixes this for me.

**Comment by marcgibbons** [comment]
Fixed in 2.0.2. Thanks!
